script/bootstrap

```diff
@@ -7,7 +7,22 @@ set -e
 
 cd "$(dirname "$0")/.."
 
+echo "==> Installing Ruby…"
```

Comment:
I'm weary of this `echo`, because...while the other steps echo what they're doing, they actually ARE doing it. When they complete fast, it tells the user that (and it's because) "oh cool, it did it and its done fast" or "it didn't have to do anything." 

Seeing...
    Installing Ruby...
...every time isn't just scary, I think it's confusing.